src/core/crypto/key_manager.cpp

```diff
@@ -398,7 +398,8 @@ static bool ValidCryptoRevisionString(std::string_view base, size_t begin, size_
 }
 
 void KeyManager::LoadFromFile(const std::string& filename, bool is_title_keys) {
-    std::ifstream file(filename);
+    std::ifstream file;
+    OpenFStream<std::ifstream>(file, filename, std::ios_base::in);
```

Comment:
This can be simplified to:

```suggestion
    OpenFStream(file, filename, std::ios_base::in);
```

The type will be auto-deduced by the template.